scout/server/blueprints/variants/templates/variants/cancer-variants.html

```diff
@@ -57,8 +57,8 @@
               <th>ExAC</th>
               <th>Type</th>
               <th>Consequence</th>
-              <th>Tumor</th>
-              <th>Normal</th>
+              <th data-toggle="tooltip" data-placement="top" title="Tumor alt. allele frequency. &#013; Alt. allele count | Ref allele count">Tumor</th>
+              <th data-toggle="tooltip" data-placement="top" title="Normal alt. allele frequency. &#013; Alt. allele count | Ref. allele count">Normal</th>
```

Comment:
I think it's better if the text is a bit shorted.

```suggestion
              <th data-toggle="tooltip" data-placement="top" title="Tumor alt. AF. &#013; Alt. allele count | Ref. allele count">Tumor</th>
              <th data-toggle="tooltip" data-placement="top" title="Normal alt. AF. &#013; Alt. allele count | Ref. allele count">Normal</th>
```